Signal/src/ViewControllers/MessageDetailViewController.swift

```diff
@@ -442,7 +442,8 @@ class MessageDetailViewController: OWSViewController, UIScrollViewDelegate {
         let contentType = attachment.contentType
         if let dataUTI = MIMETypeUtil.utiType(forMIMEType: contentType) {
             let attachment = SignalAttachment.attachment(dataSource: dataSource, dataUTI: dataUTI, imageQuality: .original)
-            let mediaMessageView = MediaMessageView(attachment: attachment, mode: .small)
+            let mediaMessageView = MediaMessageView(attachment: attachment, mode: .small, mediaDetailPresenter: self)
```

Comment:
In the hotfix, we passed the `attachmentStream:viewItem` instead of `mediaDetailPresenter:self`